app/scripts/lib/oauth-client.js

```diff
@@ -48,6 +49,10 @@ define(function (require, exports, module) {
       return this._request('get', GET_CLIENT + id);
     },
 
+    getClientKeyData: function getClientKeyData(params) {
```

Comment:
Can you add a JSDoc to this function?

Also, you should be able to use shorthand here and allow babel to "Do the Right Thing" to ensure the function name is `getClientKeyData`:

```js
getclientKeyData (params) {
   ...
},
```